sdks/wasm/package-update/README.md

```diff
@@ -0,0 +1,18 @@
+# Instructions for running
+
+Make sure that you have the latest Blazor WebAssembly version:
+
+`dotnet new -i Microsoft.AspNetCore.Components.WebAssembly.Templates::3.2.0-preview2.20160.5`
```

Comment:
I'd be cautious about saying to have the latest version and then hard-coding one in your sample command.